helmfile/values/fluentd-sc.yaml.gotmpl

```diff
@@ -1,196 +1,48 @@
-fullnameOverride: fluentd
 
-resources: {{- toYaml .Values.fluentd.resources | nindent 2  }}
-nodeSelector: {{- toYaml .Values.fluentd.nodeSelector | nindent 2  }}
-affinity: {{- toYaml .Values.fluentd.affinity | nindent 2  }}
-tolerations: {{- toYaml .Values.fluentd.tolerations | nindent 2  }}
 
-# Default args include "-q" which sets loglevel to warning.
-fluentdArgs: "--no-supervisor -q"
-configMaps:
-  useDefaults:
-    systemConf: false
-    containersInputConf: false
-    outputConf: false
-
-additionalPlugins:
-  #New version due to https://github.com/fluent/fluent-plugin-prometheus/issues/119
-  #default: gem 'fluent-plugin-prometheus', '~>1.4.0'
-- name: fluent-plugin-prometheus
-  version: 1.6.1
-
-env:
-  LIVENESS_THRESHOLD_SECONDS: 900
-  STUCK_THRESHOLD_SECONDS: 1200
-
-serviceMonitor:
+#log aggregator are daemons that continuously receives logs from log forwarders and push the log to external storage (in our case s3)
+aggregator:
+  configMap: fluentd-aggregator-configmap
+
+  replicaCount: 1
+  resources: {{- toYaml .Values.fluentd.aggregator.resources | nindent 4  }}
+  nodeSelector: {{- toYaml .Values.fluentd.aggregator.nodeSelector | nindent 4  }}
+  affinity: {{- toYaml .Values.fluentd.aggregator.affinity | nindent 4  }}
+  tolerations: {{- toYaml .Values.fluentd.aggregator.tolerations | nindent 4  }}
+
+  # Persist buffer so that the last chunks of logs can be retrieved in case of a
+  # disaster.
+  persistence:
+    enabled: true
+    size: 10Gi
+
+  extraEnv:
+   - name: AWS_ACCESS_KEY_ID
+     valueFrom:
+       secretKeyRef:
+         name: s3-credentials
+         key: s3_access_key
+   - name: AWS_ACCESS_SECRET_KEY
+     valueFrom:
+       secretKeyRef:
+         name: s3-credentials
+         key: s3_secret_key
```

Comment:
Shouldn't these only be set when `objectStorage.type: s3`?
https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/fluentd-aggregator.yaml.gotmpl#L37-L49